src/components/PlayerProfile.vue

```diff
@@ -162,15 +162,66 @@ export default {
     $route: 'executeQuery',
   },
   methods: {
+    fromNow(date) {
```

Comment:
I appreciate the effort, but I don't think re-implementing something other libraries already do a great job with is a good idea. This way it's much more error prone and there is no test coverage.